### DISCUSSION TITLE
How to exit gracefully

### DISCUSSION THREAD

**Discussion by zhangdong1015** [tchiotludo/akhq#1164] [General]
How to exit akhq gracefully

**Comment by tchiotludo** [comment]
Not sure to understand, but maybe send a sig message?

**Reply by zhangdong1015** [reply]
yes

**Reply by zhangdong1015** [reply]
maybe  it not support  sig

**Reply by tchiotludo** [reply]
As I know, Akhq support sig signals

**Reply by zhangdong1015** [reply]
but  no response with SIGTERM.
